canvas_modules/common-canvas/src/common-properties/panels/text-panel/text-panel.scss

```diff
@@ -15,7 +15,9 @@
  */
 
 .properties-text-panel {
-	padding: $spacing-04 0 $spacing-04 0;
+	:not(:empty) {
```

Comment:
When with this take effect?  I'm wondering if we should have this or not.

Reply:
This condition doesn't add spacing when text-panel is empty. 
For example, in `panelConditions_paramDef.json` -

:not(:empty) condition added -
<img width="550" alt="Screen Shot 2021-07-29 at 1 53 59 PM" src="https://user-images.githubusercontent.com/25124000/127565065-fd706de8-98fb-47b5-acc9-a8754d062d9c.png">

:not(:empty) condition removed -
<img width="616" alt="Screen Shot 2021-07-29 at 1 57 10 PM" src="https://user-images.githubusercontent.com/25124000/127565067-0b9f5d67-ad8e-4eb5-aa01-503ef9f9db0b.png">